gdk-pixbuf wants gettext >= 0.19

Building gdk-pixbuf fails because gettext is too old (v0.18.3 on Ubuntu 14.04):

```
make[1]: Entering directory `/home/djcj/mxe'
make[1]: warning: jobserver unavailable: using -j1.  Add `+' to parent make rule.
uname -a
Linux djcj 4.2.0-35-generic #40~14.04.1-Ubuntu SMP Fri Mar 18 16:37:35 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
git log --pretty=tformat:"%H - %s [%ar] [%d]" -1
64e5325364f1e14bb7281da9a0a1fff8b789b0f5 - Merge pull request #1328 from rkanavath/fix_libsvm [3 days ago] [ (grafted, HEAD, origin/master, origin/HEAD, master)]
lsb_release -a 2>/dev/null || sw_vers 2>/dev/null || true
Distributor ID: Ubuntu
Description:    Ubuntu 14.04.4 LTS
Release:    14.04
Codename:   trusty
autoconf --version 2>/dev/null | head -1
autoconf (GNU Autoconf) 2.69
automake --version 2>/dev/null | head -1
automake (GNU automake) 1.14.1
python --version
Python 2.7.6
perl --version 2>&1 | head -3

This is perl 5, version 18, subversion 2 (v5.18.2) built for x86_64-linux-gnu-thread-multi
(with 44 registered patches, see perl -V for more detail)
rm -rf   '/home/djcj/mxe/tmp-gdk-pixbuf-i686-w64-mingw32.static'
mkdir -p '/home/djcj/mxe/tmp-gdk-pixbuf-i686-w64-mingw32.static'
cd '/home/djcj/mxe/tmp-gdk-pixbuf-i686-w64-mingw32.static' &&       xz -dc '/home/djcj/mxe/pkg/gdk-pixbuf-2.32.3.tar.xz' | tar xf -
cd '/home/djcj/mxe/tmp-gdk-pixbuf-i686-w64-mingw32.static/gdk-pixbuf-2.32.3'
(cd '/home/djcj/mxe/tmp-gdk-pixbuf-i686-w64-mingw32.static/gdk-pixbuf-2.32.3' && patch -p1 -u) < /home/djcj/mxe/src/gdk-pixbuf-1-fixes.patch
patching file gdk-pixbuf/gdk-pixbuf-io.c
Hunk #1 succeeded at 193 (offset -37 lines).
patching file gdk-pixbuf-2.0.pc.in
patching file configure.ac
Hunk #1 succeeded at 136 (offset 5 lines).
cd '/home/djcj/mxe/tmp-gdk-pixbuf-i686-w64-mingw32.static/gdk-pixbuf-2.32.3' && autoreconf -fi -I'/home/djcj/mxe/usr/i686-w64-mingw32.static/share/aclocal'
autopoint: *** The AM_GNU_GETTEXT_VERSION declaration in your configure.ac
               file requires the infrastructure from gettext-0.19 but this version
               is older. Please upgrade to gettext-0.19 or newer.
autopoint: *** Stop.
autoreconf: autopoint failed with exit status: 1
make[1]: *** [build-only-gdk-pixbuf_i686-w64-mingw32.static] Error 1
make[1]: Leaving directory `/home/djcj/mxe'

real    0m4.055s
user    0m3.872s
sys 0m0.096s
```

Maybe a native build of gettext should be installed into the Linux prefix?

--- a/plugins/native/trusty/gettext.mk
+++ b/plugins/native/trusty/gettext.mk
@@ -1,0 +1,1 @@
+../gettext.mk

--- a/plugins/native/trusty/libiconv.mk
+++ b/plugins/native/trusty/libiconv.mk
@@ -1,0 +1,1 @@
+../libiconv.mk

